lld/test/ELF/lto/emit-llvm.ll

```diff
@@ -9,11 +9,13 @@
 ; RUN: ld.lld --plugin-opt=emit-llvm -mllvm -bitcode-flush-threshold=0 -o /dev/null %t.o
 ; RUN: ld.lld --lto-emit-llvm -mllvm -bitcode-flush-threshold=0 -o /dev/null %t.o
 
-; CHECK: define internal void @main()
+; CHECK: define hidden void @main()
 
 target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
 target triple = "x86_64-unknown-linux-gnu"
 
-define void @main() {
+@llvm.compiler.used = appending global [1 x ptr] [ptr @main], section "llvm.metadata"
```

Comment:
The test doesn't change for PostInternal / PreCodegen. Use a different one? Perhaps two modules are needed to show a difference.

Reply:
It optimizes it out completely, so I just made it used so it sticks around. Could make it slightly more complicated if you want.